daisy_integration_tests/image_import_and_translate_ova_with_spaces_in_vmdk.wf.json

```diff
@@ -0,0 +1,78 @@
+{
+  "Name": "image-import-and-translate-ova-with-spaces-in-vmdk",
+  "Sources": {
+    "post_translate_test.sh": "./scripts/post_translate_test.sh"
+  },
+  "Vars": {
+    "image": {
+      "Value": "image-${ID}"
+    },
+    "file": {
+      "Value": "gs://compute-image-tools-test-resources/ubuntu-1804-vmdk-spaces.ova"
+    }
+  },
+  "Steps": {
+    "import-image": {
+      "Timeout": "2h",
+      "IncludeWorkflow": {
+        "Path": "../daisy_workflows/image_import/import_and_translate.wf.json",
+        "Vars": {
+          "image_name": "${image}",
+          "source_disk_file": "${file}",
+          "translate_workflow": "ubuntu/translate_ubuntu_1804.wf.json"
+        }
+      }
+    },
+    "test-the-image": {
+      "CreateInstances": [
+        {
+          "disks": [
+            {
+              "initializeParams": {
+                "sourceImage": "${image}"
+              }
+            }
+          ],
+          "machineType": "n1-standard-4",
+          "name": "test-the-image-instance",
+          "StartupScript": "post_translate_test.sh"
+        }
+      ]
+    },
+    "wait-for-test": {
+      "Timeout": "30m",
+      "WaitForInstancesSignal": [
+        {
+          "Name": "test-the-image-instance",
+          "SerialOutput": {
+            "Port": 1,
+            "SuccessMatch": "PASSED:",
+            "FailureMatch": "FAILED:",
+            "StatusMatch": "STATUS:"
+          }
+        }
+      ]
+    },
+    "cleanup": {
+      "DeleteResources": {
+        "Images": [
+          "${image}"
+        ],
+        "Instances": [
+          "test-the-image-instance"
+        ]
+      }
+    }
+  },
+  "Dependencies": {
```

Comment:
nit: it's common to have dependencies sorted in a chronological order to make it easier to figure out ordering of steps. E.g. "test-the-image": ["import-image"] would be the first in this list, so it's easy to know that "import-image" happens first.